### ISSUE TITLE
Issues with nested folders

### ISSUE THREAD

**Issue by edahlseng** [mschuchard/linter-terraform-syntax#15]
I'm seeing an issue when using this package on nested folders. Specifically, I have a Terraform folder nested a few folders deep in a project. I can see linting errors from this package in the linter panel, but they do not show up inline in the file. The "File" column in the panel lists `fileBaseName.tf`, whereas when editing nested files in other languages this column shows `path/to/fileBaseName.tf`, which is what leads me to suspect the issue with nested folders.

**Comment by edahlseng** [comment]
Linter Debug info:
```
Platform: darwin
Atom Version: 1.41.0
Linter Version: 2.3.1
Opened file is ignored: No
Matching Linter Providers: 
  - Terraform
Disabled Linter Providers: 
Standard Linter Providers: 
  - [others excluded for brevity]
  - Terraform
UI Providers: 
  - Linter
Ignore Glob: **/*.min.{js,css}
VCS Ignored Paths are excluded: true
Current File Scopes: 
  - *
  - source.terraform
```
Terraform version: `v0.12.6`

**Comment by mschuchard** [comment]
This is an issue with the Atom-Linter dependency. It does path munging/relativity independent of whether it should or not. If you go with the normal mapping of Atom project<-->git repo<-->Terraform module, then this issue disappears. It also is not present when the module contains nested submodules (although of course that is not really optimal design/best practices, but it still works).

**Comment by mschuchard** [comment]
Likely fixed by #17. Feel free to comment if not true.
